Library/Formula/qbs.rb

```diff
@@ -1,8 +1,10 @@
 class Qbs < Formula
   desc "Qt Build Suite"
   homepage "https://wiki.qt.io/Qt_Build_Suite"
-  url "https://download.qt.io/official_releases/qbs/1.4.2/qbs-src-1.4.2.tar.gz"
-  sha256 "b9d36118c3ae0f7d4df6bf7239a0a0163c0340b701d00191fa5f832cef341ce5"
+  version "1.4.3"
```

Comment:
So basically, according to the build bot, I can't specify `version` to reduce redundancy, because it's already the same as what's determined from the URL which is constructed FROM THIS?

How can I specify the version without typing it twice? This is annoying.